src/main/legacyJava/net/fabricmc/loader/util/UrlUtil.java

```diff
@@ -28,10 +27,6 @@ public final class UrlUtil {
 	private UrlUtil() { }
 
 	public static Path asPath(URL url) throws UrlConversionException {
-		try {
-			return net.fabricmc.loader.impl.util.UrlUtil.asPath(url);
-		} catch (URISyntaxException e) {
-			throw new UrlConversionException(e);
-		}
+		return net.fabricmc.loader.impl.util.UrlUtil.asPath(url);
```

Comment:
It would be better if this caught the `WrappedException` and rethrew it as a `UrlConversionException`, any handling for invalid `URL`s won't be expecting a `RuntimeException` instead

Reply:
adjusted accordingly :)